contrib/kubernetes/datahub/charts/datahub-ingestion/README.md

```diff
@@ -0,0 +1,25 @@
+datahub-ingestion
```

Comment:
Can we name this to make it clear that it is a CronJob based ingestion? We will want to differentiate between CronJob based ingestion vs Airflow based ingestion vs any other mechanisms for scheduling metadata ingestion.

Reply:
Good point, do you having any suggestions?

Reply:
Something as simple as "datahub-ingestion-cronjob" or "datahub-ingestion-scheduler-cronjob" would do? WDYT

Reply:
shorter names plz: "datahub-ingestion-cron" :)